yarn-project/ethereum/src/test/rollup_cheat_codes.ts

```diff
@@ -194,6 +192,29 @@ export class RollupCheatCodes {
     });
   }
 
+  /**
+   * Sets the archive for a specific block number in the rollup storage.
+   * @param blockNumber - The block number to set the archive for
+   * @param archive - The archive value
+   */
+  public async setArchiveForBlock(blockNumber: number, archive: `0x${string}`) {
```

Comment:
Should this not be called on its own with `markAsProven` to avoid manually needing to do it? Seems fairly error prone without it being in there.